### PR TITLE
Encoder.detach() method

### PR DESCRIPTION
**Pull request by KSSS1**
Including a method for detaching() the attached encoders. Usefull for preventing "too many encoders.FAIL" error when switching directions. Or just to disable encoders on the fly.

### CODE REVIEW

**Comment by madhephaestus** [comment]
This is the wrong branch to send PR's from, thats the website and should not be merged into master.
